crates/zoon/src/style/spacing.rs

```diff
@@ -1,11 +1,26 @@
 use crate::*;
 
+/// Define space between elements.
+/// More information at <https://developer.mozilla.org/en-US/docs/Web/CSS/gap>.
```

Comment:
Hei, Why do we need extra lines for space ?  I saw some other commit suggestion that I resolved because I thought it was old/ outdated code comparison.

Reply:
At least Rust Analyzer displays it as `Define space between elements. More information ...`.
It's a bit more readable with spaces around it in the code / as multiline in IDEs or docs imho.
Just cosmetic change. (Correct me if it's rendered as multiline in both docs and RA.)

Reply:
Never mind I answered in the other discussion.

Reply:
Okay, I see that sometimes it does make a difference, and sometimes it does not and set as much as possible on one line.

For the other discussion, I did not see any

I found one example where there is little difference.

![with_extra_lines](https://user-images.githubusercontent.com/30114057/162015287-e6d26d64-46e8-4570-b666-0d1701e52eb8.png)
![without_lines](https://user-images.githubusercontent.com/30114057/162015326-70073027-fec9-4084-939c-190e171703a8.png)


But the space around #Example is always ignored from what I see. The `md` headers are highlighter in Jetbrain, so adding extra space is not useful.

Reply:
VS Code is the most used tool among Rust devs and Rust Analyzer is officially a part of the Rust organization so I would primarily focus on this combination. I've also migrated to VS Code from Intellij several years ago so when I say something doesn't work in IDE I probably think of VS Code.

Yes, spaces around `# Example` are not really needed but it looks much better in the code imho. Also it's written this way in official docs: https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#making-useful-documentation-comments.

And it's recommended in general to write shorter lines because they are easier to scan and read (not only in development but also in content copywriting).